src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -104,7 +104,7 @@ const AdminDashboard = () => {
   useEffect(() => {
     axios
       .get(
-        'https://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/to-approve'
```

Comment:
Got confirmation that the deployed DS endpoints via swagger are HTTP not HTTPS, so this is fine